### ISSUE TITLE
onLoadMoreAsync isn't called until the scroll movement even if bottom is visible 

### ISSUE THREAD

**Issue by agolovenko** [exponentjs/react-native-infinite-scroll-view#9]
Well, this one is more of a feature request.

Say my async load function only fetches 3 rows at a time. And 3 rows are just half of the screen. 
For now I need to scroll the rows in order onLoadMoreAsync to be triggered to load more rows. Ideally I'd like it to be triggered while (canLoadMore && the screen isn't full with rows). 

Otherwise great project, thanks=)

**Comment by Jaafar-abusair** [comment]
show us your code dude

**Comment by ide** [comment]
Yeah, this is a real bug. I spent some time a few months ago trying to fix it cleanly but got stuck since the notion of rows is specific to ListViews, not ScrollViews, so InfiniteScrollView doesn't have enough information to know when ListView has finished rendering the rows (potentially after several frames, depending on what your `pageSize` prop is set to) and that we need to fetch more. Sorry I don't have a more optimistic update than that.

**Comment by Jaafar-abusair** [comment]
@ide what about the loading indicator? Its not working for me

**Comment by ide** [comment]
I think that's unrelated to this issue.
